tests/test_seasonal_water_yield_regression.py

```diff
@@ -1260,17 +1260,221 @@ def test_local_recharge_undefined_nodata(self):
 
         # arbitrary values for alpha, beta, gamma, etc.
         # not verifying the output, just making sure there are no errors
+        try:
+            seasonal_water_yield_core.calculate_local_recharge(
+                [precip_path for i in range(12)], [et0_path for i in range(12)],
+                [quickflow_path for i in range(12)], flow_dir_path,
+                [kc_path for i in range(12)], {i: 0.5 for i in range(12)}, 0.5,
+                0.5, stream_path,
+                os.path.join(self.workspace_dir, 'target_li_path.tif'),
+                os.path.join(self.workspace_dir, 'target_li_avail_path.tif'),
+                os.path.join(self.workspace_dir, 'target_l_sum_avail_path.tif'),
+                os.path.join(self.workspace_dir, 'target_aet_path.tif'),
+                os.path.join(self.workspace_dir, 'target_precip_path.tif'))
+        except Exception as e:
+            assert False, f"calculate_local_recharge raised an exception: {e}"
+
+    def test_calculate_local_recharge(self):

```

Comment:
Since the setup in this test is almost identical to `test_local_recharge_undefined_nodata` do you think it would make sense to combine them? Basically taking the assertions you added in this test and applying them to the pre-existing test that had no assertions?